### DISCUSSION TITLE
Saving anvil world while forks haven't been completed

### DISCUSSION THREAD

**Discussion by CDX-1** [Minestom/Minestom#2419] [Help]
How could I get around this issue:
![image](https://github.com/user-attachments/assets/65548461-2462-40b9-8227-5dfc8910ab06)
For more context as to what's going on, I'm using generator forks to place schematics in the world generator. However, if the world saves before the schematic is done placing, it'll leave a chunk of air where the rest of the schematic was supposed to go.

**Comment by kermandev** [comment]
I think of two ways to solve this, It would to be to utilize a CountdownLatch the world save on your placing of schematics. (So you wait for the forks to finish and unblock the saving thread). Or you could just manually save/queue the affected world after paste. Depends on your utilization.
